src/lib/Scenes/Collection/Collection.tsx

```diff
@@ -25,8 +29,12 @@ interface CollectionState {
 export class Collection extends Component<CollectionProps, CollectionState> {
   state = {
     sections: [],
+    isArtworkGridVisible: false,
+    isFilterArtworksModalVisible: false,
+  }
+  viewabilityConfig = {
+    viewAreaCoveragePercentThreshold: 30, // 30% of the artworks component should be in the screen before toggling the filter button
```

Comment:
(no action expected) - I appreciate long names, and I usually expect them to fulfill the need of a comment. In this case, we have both - I wonder if it means the name isn't quite telling the story?

Reply:
In a previous review it was suggested that comment be added, https://github.com/artsy/emission/pull/2068#discussion_r374319787.

But I agree that it is pretty clear from the name that a comment probably isn't necessary.

Reply:
The `viewAreaCoveragePercentThreshold` is part of react native's [FlatList API](https://facebook.github.io/react-native/docs/flatlist#viewabilityconfig)! But you're right, it's a little confusing. 😄 The comment helps, thank you!